src/shared/interfaces.ts

```diff
@@ -55,6 +55,7 @@ export interface WritableWorkspaceConfig {
 	flutterSdkHome?: string;
 	flutterSyncScript?: string;
 	flutterTestScript?: CustomScript;
+	flutterToolsScript?: CustomScript;
```

Comment:
Is this intended to be used for all Flutter invocations (for example when we run "flutter pub get" or "flutter doctor")? If not, is it better named as `flutterDebugAdapterScript` or similar? (and if so, do we need to do that now?)

Reply:
This is a little confusing. There are some flutter invocations that can share this, but not all of them. I can change over the ones that can use it in a later change, but this script isn't specific to debugging.

Reply:
SGTM. It may be worth a comment about which it can/should be used for so we have it written here somewhere in this codebase (for my benefit - or others that might not have internal knowledge).

Reply:
Done, thanks!